### PR TITLE
Added first generation of script to generate mole fractions

### PR DESCRIPTION
**Pull request by pgrinaway**
This is the really basic script that I had used previously to generate the list of masses. It takes a `yaml` input (example provided). A couple things to note:
- The format of the `yaml` file could possibly be improved
- We want to have the script output an importable list of samples
  - The way that I had conceived of this a long time ago was probably not the best (writing out the undocumented format of the internal database)

Since this script does work, though, I'd say we might want to merge soon (perhaps I should add a test after lunch) and work on these improvements later.

### CODE REVIEW

**Comment by jchodera** [comment]
Thanks! Can you add a `README.md` showing how to run it?

**Comment by jchodera** [comment]
Also, how about putting it in `scripts/quantos-dosing-series/` or something a bit more descriptive?

**Comment by jchodera** [comment]
Also, a sample output would be good!  That can go into the `README.md` if it's just printed to `stdout`.

**Comment by pgrinaway** [comment]
Added example output, renamed directory, added `README.md`. Let me know if something else is needed before merging.

File: scripts/quantos-dosing-series/density_output_example.txt [hunk 1]
```diff
@@ -0,0 +1,12 @@
+DMSO,H2O
+-0.000000,10.000000
```
Comment (jchodera) [inline scripts/quantos-dosing-series/density_output_example.txt]: Negative zero?  :)

File: scripts/quantos-dosing-series/example.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+compound1_name : DMSO
+compound2_name : H2O
+n_fractions : 11
+total_mass : 12
```
Comment (jchodera) [inline scripts/quantos-dosing-series/example.yaml]: Are comments tolerated?  If so, might be good to note `# grams` after `total_mass` and `# grams/mol` after `mw`

**Comment by jchodera** [comment]
Other than minor comments in input example, looks good to me!  Thanks!

**Comment by pgrinaway** [comment]
Added comments to the example. Also, I agree that negative zero is kind of wonky, I think it has to do with the IEEE specification of floating point numbers. I'm going to try to get rid of that (I am concerned that LabX may not read it in correctly).

**Comment by pgrinaway** [comment]
Ok, I've done `np.abs()` to remove the -0. I don't think that should cause any issues, since it's just taking the absolute value of something that shouldn't be negative as it's being printed.

**Comment by jchodera** [comment]
Go ahead and merge!

**Pull request merged**